source/plugins/ruby/in_kube_nodes.rb

```diff
@@ -157,6 +155,19 @@ def parse_and_emit_records(nodeInventory, batchTime = Time.utc.iso8601)
         insightsMetricsEventStream = MultiEventStream.new
         kubePerfEventStream = MultiEventStream.new
         @@istestvar = ENV["ISTEST"]
+        
+        begin
+          # first refresh the node cpu and memory capacity cache
```

Comment:
This will be invoked for every min and its not first refresh, right?

Reply:
Correct, it will be invoked every minute (we don't know when new nodes will be added, so I think it needs to be refreshed every minute)